src/openforms/forms/migrations/0076_move_confirmation_email_setting.py

```diff
@@ -0,0 +1,55 @@
+# Generated by Django 3.2.18 on 2023-04-14 08:38
+
+from django.db import migrations
+
+from openforms.forms.constants import ConfirmationEmailOptions
+
+
+def forward(apps, schema_editor):
+    """Change the setting for the confirmation email from a choices field to a boolean field"""
+    Form = apps.get_model("forms", "Form")
+
+    forms = Form.objects.all()
+
+    for form in forms:
+        form.send_confirmation_email = (
+            form.confirmation_email_option != ConfirmationEmailOptions.no_email
+        )
+
+    Form.objects.bulk_update(forms, fields=["send_confirmation_email"])
+
+
+def backwards(apps, schema_editor):
+    """Change the setting for the confirmation email from a boolean field to a choices field."""
+    Form = apps.get_model("forms", "Form")
+
+    forms = Form.objects.all().select_related("confirmation_email_template")
+    for form in forms:
+        if not form.send_confirmation_email:
+            form.confirmation_email_option = ConfirmationEmailOptions.no_email
+            continue
+
+        # Does it have a custom template or is the custom template usable?
+        if hasattr(form, "confirmation_email_template") and bool(
+            form.confirmation_email_template.subject
+            and form.confirmation_email_template.content
+        ):
+            form.confirmation_email_option = (
+                ConfirmationEmailOptions.form_specific_email
+            )
+            continue
+
+        form.confirmation_email_option = ConfirmationEmailOptions.global_email
```

Comment:
the reverse can also be done with a single Case/When query I think

Reply:
I couldn't get this one to work with one query :( I added an alternative, not sure if it's better

Reply:
worksforme